packages/arb-token-bridge-ui/src/components/common/Dialog.tsx

```diff
@@ -98,10 +101,22 @@ export function Dialog(props: DialogProps) {
             props.children
           ) : (
             <>
-              <div className="px-8 py-4">
+              <div className="relative px-8 py-4">
                 <HeadlessUIDialog.Title className="text-2xl font-medium">
                   {props.title}
                 </HeadlessUIDialog.Title>
+                {props.closeable && (
+                  <button
+                    type="button"
+                    className="absolute top-5 right-10"
```

Comment:
Instead of setting the positioning to absolute, let's try to do it with a flex container, so something like:

```
<div className="flex items-center justify-between px-8 py-4">
  <HeadlessUIDialog.Title className="text-2xl font-medium">
    {props.title}
  </HeadlessUIDialog.Title>
  <button />
</div>  
```